### PR TITLE
feat: Redesign Saved Schedules UI + Fix filtering by term

### PR DESCRIPTION
**Pull request by jaskirat-gill**

### CODE REVIEW

File: src/content/Settings/WorklistActions/SyncSavedSchedules/SyncSavedSchedules.tsx [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+            <div
+              className="instructions-info-button"
+              onClick={dispatchInstructions}
+            >
+              <QuestionIcon size={16} />
+            </div>
+          </div>
+          <div className="worklist-container">
+            <div>Worklist: </div>
+            {[0, 1, 2, 3].map((worklist) => (
```
Comment (ThatNerdSquared) [inline src/content/Settings/WorklistActions/SyncSavedSchedules/SyncSavedSchedules.tsx:74]: can you not use `TabBar` here? would avoid the duplication of styles as well

File: src/content/modalBodies/Manual Entry/ManualEntryModalBody.css
Comment (ThatNerdSquared) [inline src/content/modalBodies/Manual Entry/ManualEntryModalBody.css:1]: can we not put these in a folder? i feel like it's pretty unnecessary, the deep nesting is inconvenient and doesn't add anything organizationally IMO

Comment (jaskirat-gill) [reply src/content/modalBodies/Manual Entry/ManualEntryModalBody.css:1]: Thats mb, left over from when i had sync schedule modal bodies. then this folder had like 5 unrelated files so the folders made sense. Ill revert this

File: src/content/Settings/WorklistActions/SyncSavedSchedules/SyncSavedSchedules.tsx (hunk 1 above)
Comment (jaskirat-gill) [reply src/content/Settings/WorklistActions/SyncSavedSchedules/SyncSavedSchedules.tsx:74]: I did not know about that component lol. will try that

**Pull request merged**